Debugging with backlash does not work in Python3

When running a pecan application with Python3, if you install the PyPI backlash package (https://pypi.python.org/pypi/backlash) for interactive debugging and encounter an exception, you'll see:

    [ERROR] Error handling request /
    Traceback (most recent call last):
    ...
      File "/path/to/lib/python3.4/site-packages/pecan/middleware/debug.py", line 23, in debug_application
        yield self.body_re.sub('\g<1>%s' % __CONFIG_HELP__, part)
    TypeError: can't use a string pattern on a bytes-like object

--- a/pecan/middleware/debug.py
+++ b/pecan/middleware/debug.py
@@ -10,17 +10,18 @@ __CONFIG_HELP__ = '''
 
 try:
     import re
+    from six import b
     from backlash.debug import DebuggedApplication
 
     class DebugMiddleware(DebuggedApplication):
 
-        body_re = re.compile('(<body[^>]*>)', re.I)
+        body_re = re.compile(b('(<body[^>]*>)'), re.I)
 
         def debug_application(self, environ, start_response):
             for part in super(DebugMiddleware, self).debug_application(
                 environ, start_response
             ):
-                yield self.body_re.sub('\g<1>%s' % __CONFIG_HELP__, part)
+                yield self.body_re.sub(b('\g<1>%s' % __CONFIG_HELP__), part)
 
 
 except ImportError:

